dev_guide/cron_jobs.adoc

```diff
@@ -70,14 +70,79 @@ spec:
   jobTemplate:             <2>
     spec:
       template:
+        metadata:
+          labels:          <3>
+            parent: "cronjobpi"
         spec:
           containers:
           - name: pi
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: Never
+          restartPolicy: OnFailure <4>
 ----
 
-1. Schedule for the job. In this example, a job will run every minute.
+1. Schedule for the job. In this example, the job will run every minute.
 2. Job template. This is similar to the xref:../dev_guide/jobs.adoc#creating-a-job[job example].
+3. Sets a label for jobs spawned by this cron job. 
+4. The restart policy of the pod. This does not apply to the job controller. See xref:../dev_guide/jobs.adoc#creating-a-job-known-issues[Known Issues and Limitations] for details.
 ====
+
+Alternatively, you can create and launch a cron job from a single command -- specifically, `oc run`. The following command creates and launches the same cron job as the `pi` resource:
+
+----
+$ oc run pi --image=perl --schedule='*/1 * * * *' --restart=OnFailure --labels parent="cronjobpi" --command -- perl -Mbignum=bpi -wle 'print bpi(2000)' 
```

Comment:
Its a long command and should be broken in multiple lines, so that it renders correctly. (https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks)